create-integration-deployment/action.yml

```diff
@@ -0,0 +1,71 @@
+name: Create integration deployment PR
```

Comment:
This is the github action that must be reviewed with detail. This is actually in charge of creating a PR in the integration-deployments repository, the real goal of the ticket in question.